region/maridia/outer.json

```diff
@@ -120,6 +120,128 @@
+                      "artificialMorph": false,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken",
+                    {"or":[
+                      "Gravity",
+                      "HiJump",
+                      "canSpringBallJumpMidAir"
+                    ]}
+                  ]
+                }
+              ]
+            },
+            {
+              "leavesWithArtificialMorph": true,
+              "strats": [
+                {
+                  "name": "Carry G-Mode Morph Through Tube (Right to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [3],
+                      "artificialMorph": true,
+                      "mode": "any"
+                    }},
+                    {"or":[
+                      "Morph",
+                      "SpringBall",
+                      {"and":[
+                        "Gravity",
+                        "h_canArtificialMorphIBJ"
+                      ]},
+                      {"and":[
+                        "Gravity",
+                        "h_canArtificialMorphBombHorizontally",
+                        {"or": [
+                          "Bombs",
+                          {"ammo": { "type": "PowerBomb", "count": 1 }}
+                        ]}
+                      ]},
+                      {"and":[
+                        { "not": "f_MaridiaTubeBroken" },
+                        "canRiskPermanentLossOfAccess"
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Morph Through Tube (Bottom to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [2],
+                      "artificialMorph": true,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken",
+                    {"or": [
+                      {"and": [
+                        "Gravity",
+                        "h_canArtificialMorphMovement"
+                      ]},
+                      {"and": [
+                        "canSuitlessMaridia",
+                        "SpringBall",
+                        "HiJump"
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Carry G-Mode Morph Through Tube (Top to Left)",
+                  "notable": false,
+                  "requires": [
+                    {"comeInWithGMode": {
+                      "fromNodes": [4],
+                      "artificialMorph": true,
+                      "mode": "any"
+                    }},
+                    "f_MaridiaTubeBroken",
+                    {"or":[
+                      "Gravity",
+                      "HiJump"
```

Comment:
"canSpringBallJumpMidAir" is in the other `Top to Left`.  It should be here too?
Same with the artificial morph `Top to Right`

Reply:
That would require morph, and this is a duplicate from above, except this one is leaving with artificial morph. If you have morph, I _think_ that you should be able to use non-artificial morph strat above and then use morph in the next room and it will work out the same. I'll check with Maddo tomorrow

Reply:
Yeah, this is how he says it works.